src/components/ApplicationModal/ModalBasicInfo/modalBasicInfo.scss

```diff
@@ -0,0 +1,69 @@
+.info-display {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    height: auto;
+    overflow: auto;
+
+    .info-column {
+        display: flex;
+        flex-direction: column;
+        margin-right: 20px;
+
+        .info-box {
+            background-color: #0E5385;
+            box-shadow: 2px 2px rgba(0,0,0,0.25);
+            width: 100%;
+            height: auto;
+            border-radius: 5px;
+            border: 1px solid black;
+            padding: 10px 20px;
+            margin-bottom: 10px;
+            margin-top: 10px;
+            overflow: auto;
+        
+            h1 {
```

Comment:
I have to admit that I don't know exactly how CSS/SCSS deals with nested classes like this, but I know that if you just throw an `h1` into a regular css file, it assumes that we want that class to apply to _all_ `h1`'s in the app. So just check to make sure that the styling in other places isn't affected by this change! Take a look at some of the `styles.module.css` files for components and their corresponding `index.js` files to see how we've been handling styling in other places in the app.

Reply:
I think that the nested scss stylings are automatically scoped to the respective component (so no global issues) but to be perfectly honest this was the first time I've used scss as well lol, will let @wzhang2705 touch on this

Reply:
This is the difference between CSS and SCSS here. For SCSS, since h1 is defined within the class info-box, it only applies the styling to an h1 tag that sits within and .info-box class. This is also the reason why I wanted to use SCSS because you can nest classes this way, and you can't do that in plain CSS. Also, the SCSS styling should be limited to those components that use this SCSS file, and shouldn't be extended to any global styles.

Reply:
Sounds good to me!